ext/web/01_mimesniff.js

```diff
@@ -395,6 +395,10 @@ const ImageTypePatternTable = [
 
 /**
  * Ref: https://mimesniff.spec.whatwg.org/#image-type-pattern-matching-algorithm
+ * NOTE: Some browsers have implementation-defined image formats.
+ * For example, The AVIF image format is supported by all browsers today.
+ * However, the standardization seems to have hard going.
+ * See: https://github.com/whatwg/mimesniff/issues/143
  * @param {Uint8Array} input
  * @returns {string | undefined}
  */
```

Comment:
This question is unrelated to quoted the line.
I think that the `sniffImage` function should receives another argument which is a byte sequence as `Uint8Array` and passes to `imageTypePatternMatchingAlgorithm`.
It seems that the `imageTypeMatched` variable will always become `undefined` with the current implementation.

https://github.com/denoland/deno/blob/c7cba4eda73e000baa6bfbce6a156f9974edee36/ext/web/01_mimesniff.js#L418-L441

Reply:
I dont have strong opinions on this, though the spec does specify a single argument, but givne that it is internal functionality should be fine to deviate. so don't mind as long as we are compliant in behaivour